src/containers/UsersList.js

```diff
@@ -1,8 +1,105 @@
-import React, { Component } from "react";
-import { Header } from "semantic-ui-react";
-
+import React, { Component, Fragment } from "react";
+import { Header, Card, Grid } from "semantic-ui-react";
+import Paginate from "../components/Paginate";
+import PaginationLinks from "../components/PaginationLinks";
+import { connect } from "react-redux";
+import { fetchUsers } from "../actions/getUsersAction";
+import User from "../components/User";
+import "../assets/UsersList.css";
 export class UsersList extends Component {
```

Comment:
Not 100% sure about the pattern. We should use classes or the other syntax for all the component.
My preference is towards Classes, but leave it up to you

Reply:
I'm not sure I follow you... are you referring to the fact that I have used a class for the `UserList` container, but am using a function for the `User` component? I understood that since it is a presentational component a function would be sufficient. 

@Kachulio1 we largely wrote this together, what say you? Also, maybe I just misunderstood what you mean @joaopapereira

Reply:
Nope you are on point

Reply:
Personally I prefer consistency between things that are similar